apps/saved_searches/saved_searches.py

```diff
@@ -92,7 +92,8 @@ def get(self, req, lookup):
 class SavedSearchesService(BaseService):
     def on_create(self, docs):
         for doc in docs:
-            doc.setdefault('user', request.view_args.get('user'))
+            if 'user' not in doc:
```

Comment:
imo would be better to add also `and request:`, user might still not be there in data